### PR TITLE
A handler must return a promise

### PR DESCRIPTION
**Pull request by SacDeNoeuds**
when not sending result via callback

### CODE REVIEW

**Comment by SacDeNoeuds** [comment]
#### TL;DR

I think that more generally, what I don't get currently is how to set up a local environment to develop an api with scaleway functions, if you have any guide on that I'm very interested, unless I'm the only one in that situation. In which case, just close the PR :shrug: 

PS :  :heart: your avatar :wink: 

---

I retested it to make sure I'm a newbie to serverless:
When developing locally with serverless-offline (which is common from what I understand), when testing an http get the function never resolves. When I changed the function to return a promise, then the function resolved properly the response.

I also had another issue with the runtime not recognized when using serverless-offline, setting "nodejs14.x" fixed it.
I think all these issues are due to integration with serverless-offline.

On the same matter, for local development I need to register http events but when I deploy, I get an error
```
Error: "Trigger Type http is not currently supported by Scaleway's Serverless platform, supported types are the following: schedule"
```

From the [scaleway doc](https://www.scaleway.com/en/docs/compute/functions/api-cli/fun-uploading-with-serverless-framework/#configuring-events):
>  Note:
We do not include HTTP triggers in our event types, as an HTTP endpoint is created for every function. Triggers are just another way to trigger your functions, but you will always be able to execute your code via HTTP.

Since scaleway functions already has an http endpoint, why doesn't the plugin just ignore the http events?

**Comment by thomas-tacquet** [comment]
Everything seems available to https://github.com/scaleway/serverless-functions-node 

Do not hesistate to raise a new issue if it still relevant 👌  thank you for your contribution
